.github/workflows/ci-cd-gradle.yml

```diff
@@ -20,6 +20,9 @@ on:
       sonar-token:
         description: A token allowing access to SonarCloud.
         required: true
+      github-token:
+        description: The token used for rejecting runs
+        required: false
```

Comment:
Setting `required` and `false` to avoid breaking the existing deployment pipelines that use `ci-cd-gradle.yml`. Will change it to `ture` once all related repo are updated.

Reply:
This should be renamed to be explicit about what the token is and/or the description changing to summarise what the token needs permissions to do.